tools/test/README.md

```diff
@@ -39,3 +39,19 @@ You can also pass a `--config` option to the test command:
 ```bash
 > lg test --config ./path/to/config
 ```
+
+### Testing in React 17
+
+In CI we test our components using React 17.
+
+To do this manually, first run the [init script](./scripts/install-react17.ts) in order to install the specific testing dependencies needed for React 17. (`yarn init17` from the repo root)
+
+```bash
+> yarn init17
```

Comment:
to exit react17 mode, would we run `yarn`?

Reply:
I think you need to clean first, but yeah. I'll add that